### PR TITLE
Adding Krahmer2018 pcp data

### PR DESCRIPTION
**Pull request by ococrook**
Note there are 3 experiments in this dataset LFD, HFD and HFD after 12 hours. 

I could not find a marker list used!!

There is another set of phospho-enriched data to come in a seperate MSnSet.

### CODE REVIEW

File: inst/scripts/krahmer2018pcp.R [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Experiment info
+toName <- paste0(colnames(pcpcsv)[16:213])
+colnames(e) <- toName
+pd <- data.frame(toName,
+                 row.names=colnames(e))  
+pd <- new("AnnotatedDataFrame", pd)
+
+## feature data
+ascession
```
Comment (lgatto) [inline inst/scripts/krahmer2018pcp.R]: This seems to be a left-over from somewhere, as it isn't defined and results in an error.

File: inst/scripts/krahmer2018pcp.R [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+krahmer2018pcp <- new("MSnSet",
+                 exprs = e,
+                 phenoData = pd,
+                 experimentData = experiment,
+                 featureData = fd)
+
+
+krahmer2018pcp@processingData <- process
+if (validObject(krahmer2018pcp)) 
+  return (krahmer2018pcp)
```
Comment (lgatto) [inline inst/scripts/krahmer2018pcp.R]: A `return` statement is only valid inside a function. I think here you should have
```r
stopifnot(validObject(krahmer2018pc))
```

File: man/krahmer2018pcp.Rd [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  correlates with reduced hepatic protein secretion. Our systematic in vivo
+  analysis of subcellular rearrangements and organelle-specific phosphorylation
+  reveals how nutrient overload leads to organellar reorganization and cellular dysfunction.
+  
+}
+
+\usage{
+  data("krahmer2018pcp")
+}
+
```
Comment (lgatto) [inline man/krahmer2018pcp.Rd]: Could you add *format*, *references* and *examples* sections. See `hyperLOPIT2015.Rd` for an example.

File: inst/scripts/krahmer2018pcp.R (hunk 2 above)
Comment (lgatto) [reply inst/scripts/krahmer2018pcp.R]: You also need a `save` at the end of that script that generates the `rda` file in `data`.

File: inst/scripts/krahmer2018pcp.R (hunk 1 above)
Comment (ococrook) [reply inst/scripts/krahmer2018pcp.R]: Yes, not sure how the error passed me ... anyway: removed.

File: inst/scripts/krahmer2018pcp.R (hunk 2 above)
Comment (ococrook) [reply inst/scripts/krahmer2018pcp.R]: changed.

Is there a preference for creating `rda` files over `rdata` files. I'm ignorant to the difference? Anyway, I've changed to rda.

File: man/krahmer2018pcp.Rd (hunk 3 above)
Comment (ococrook) [reply man/krahmer2018pcp.Rd]: added

File: inst/scripts/krahmer2018pcp.R (hunk 2 above)
Comment (lgatto) [reply inst/scripts/krahmer2018pcp.R]: I would use either `rda` or `RData`. I then to use the latter nowadays.

**Comment by lgatto** [comment]
Thank you for the PR, Olly.

I forgot to point out two things
- update the `NEWS` file
- knit `README.Rmd` to update `README.md` with the new data
I will do this now.

**Pull request merged**